pkg/config/config.go

```diff
@@ -184,6 +184,10 @@ type ContainersConfig struct {
 	// NoHosts tells container engine whether to create its own /etc/hosts
 	NoHosts bool `toml:"no_hosts,omitempty"`
 
+	// OOMScoreAdj tunes the host's OOM preferences for containers
+	// (accepts values from -1000 to 1000).
+	OOMScoreAdj int `toml:"oom_score_adj,omitempty,omitzero"`
```

Comment:
```suggestion
	OOMScoreAdj *int `toml:"oom_score_adj,omitempty,omitzero"`
```
AFAICT 0 is a valid value, so we need a way to see a difference between value not set and set to 0 by a user.

see https://github.com/containers/podman/commit/164b64ea3baa8502a23fc0c7674f4a7e60507aa0